### PR TITLE
feat(k8s): kube cluster creation update

### PR DESCRIPTION
**Pull request by bene2k1**
### Your checklist for this pull request

- [ ] Name your pull request according to the [contribution guidelines](https://github.com/scaleway/docs-content/blob/main/docs/CONTRIBUTING.md).

### Description

Please describe what you added or changed.

### CODE REVIEW

File: containers/kubernetes/how-to/create-cluster.mdx [hunk 1]
```diff
@@ -1,53 +1,88 @@
...
+  * Simplified cluster management via [kubectl](https://kubernetes.io/docs/reference/kubectl/overview/), the Kubernetes command-line tool.
 
-To administrate your Kubernetes cluster easily, Scaleway provides a `.kubeconfig` file that allows you to manage your cluster from your local computer by using `kubectl`. [Kubectl](https://kubernetes.io/docs/reference/kubectl/overview/) is the command line interface for running commands against Kubernetes clusters.
+To facilitate cluster administration, Scaleway provides a `.kubeconfig` file, enabling you to manage your cluster locally using `kubectl`. [kubectl](https://kubernetes.io/docs/reference/kubectl/overview/) is essential for executing commands against Kubernetes clusters.
 
 <Macro id="requirements" />
 
-- A Scaleway account logged into the [console](https://console.scaleway.com)
-- [Owner](/identity-and-access-management/iam/concepts/#owner) status or [IAM permissions](/identity-and-access-management/iam/concepts/#permission) allowing you to perform actions in the intended Organization
+Prerequisites:
```
Comment (jcirinosclwy) [inline containers/kubernetes/how-to/create-cluster.mdx:27]:
```suggestion
```

File: containers/kubernetes/how-to/create-cluster.mdx [hunk 2]
```diff
@@ -1,53 +1,88 @@
...
+- A Scaleway account with active login to the [console](https://console.scaleway.com)
```
Comment (jcirinosclwy) [inline containers/kubernetes/how-to/create-cluster.mdx:28]:
```suggestion
- A Scaleway account with access to the [console](https://console.scaleway.com)
```

File: containers/kubernetes/how-to/create-cluster.mdx [hunk 3]
```diff
@@ -1,53 +1,88 @@
...
+- [Owner](/identity-and-access-management/iam/concepts/#owner) status or [IAM permissions](/identity-and-access-management/iam/concepts/#permission) for the relevant organization
```
Comment (jcirinosclwy) [inline containers/kubernetes/how-to/create-cluster.mdx:29]:
```suggestion
- [Owner](/identity-and-access-management/iam/concepts/#owner) status or [IAM permissions](/identity-and-access-management/iam/concepts/#permission) for the relevant Organization
```

File: containers/kubernetes/how-to/create-cluster.mdx [hunk 4]
```diff
@@ -1,53 +1,88 @@
...
+This section outlines the settings for your cluster pools.
+
+1. Specify the following for each pool:
+    * The **Availability Zone** for the pool's nodes.
+    * The **node type** for the pool.
+    * Configure **pool options**, including node count and whether to enable autoscaling. Options also include enabling autoheal and linking to a placement group, or you can retain default settings.
+        <Message type="tip">
+          Unsure about the autoheal feature? [Learn more about autoheal.](/containers/kubernetes/concepts/#autoheal)
+        </Message>
+    * Enable full isolation if required.
```
Comment (jcirinosclwy) [inline containers/kubernetes/how-to/create-cluster.mdx:71]:
```suggestion
    * Enable full isolation, if required.
```

File: containers/kubernetes/how-to/create-kosmos-cluster.mdx [hunk 5]
```diff
@@ -23,24 +23,66 @@ Kubernetes Kosmos provides an alternative to a classic [Kubernetes Kapsule](/con
...
-    * The **node type** you need.
-    * The configuration for your **pool options**, including the number of nodes and whether to enable autoscale. You can also choose whether to enable autoheal and whether to link the cluster to a placement group. Alternatively, you can leave these options at default values.
-6. Click **Create cluster**. Your cluster is deployed, and you are taken to the cluster’s **Overview** tab.
+    * Choose the geographical **region** for the cluster.
+    * Select the control plane offer for your cluster. Options include shared or dedicated control planes.
+        <Message type="tip">
+          Need help deciding on a control plane offer? Learn more about our [Kubernetes control plane offers](/containers/kubernetes/reference-content/kubernetes-control-plane-offers/).
+        </Message>
+    * Specify the Kubernetes **version** for your cluster.
+4. Provide a **name** for the cluster, along with an optional description and tags for better organization.
```
Comment (jcirinosclwy) [inline containers/kubernetes/how-to/create-kosmos-cluster.mdx:45]:
```suggestion
4. Provide a **name** for the cluster. Optionally, you can add a description and tags for better organization.
```

File: containers/kubernetes/how-to/create-kosmos-cluster.mdx [hunk 6]
```diff
@@ -23,24 +23,66 @@ Kubernetes Kosmos provides an alternative to a classic [Kubernetes Kapsule](/con
...
+1. Choose the pool type. Available types are **Scaleway** or **multi-cloud**.
+2. Specify the following for each pool:
+  * For **Scaleway** pools:
+    * The **Availability Zone** for the pool's nodes.
+    * The **node type** for the pool.
+    * Configure **pool options**, including node count and whether to enable autoscaling. Options also include enabling autoheal and linking to a placement group, or you can retain default settings.
+        <Message type="tip">
+          Unsure about the autoheal feature? [Learn more about autoheal.](/containers/kubernetes/concepts/#autoheal)
+        </Message>
+    * Enable full isolation if required.
```
Comment (jcirinosclwy) [inline containers/kubernetes/how-to/create-kosmos-cluster.mdx:68]:
```suggestion
    * Enable full isolation, if required.
```

File: containers/kubernetes/how-to/create-kosmos-cluster.mdx [hunk 7]
```diff
@@ -23,24 +23,66 @@ Kubernetes Kosmos provides an alternative to a classic [Kubernetes Kapsule](/con
...
+        <Message type="tip">
+          Need more information on full isolation? [Learn more about full isolation.](/containers/kubernetes/reference-content/secure-cluster-with-private-network/#what-is-the-difference-between-controlled-isolation-and-full-isolation)
+        </Message>
+  * For **multi-cloud** pools:
+    * TBD.
+2. Click **Add pool** to integrate the pool into the cluster.
+
```
Comment (jcirinosclwy) [inline containers/kubernetes/how-to/create-kosmos-cluster.mdx:75]:
```suggestion
```

File: containers/kubernetes/how-to/create-kosmos-cluster.mdx [hunk 8]
```diff
@@ -23,24 +23,66 @@ Kubernetes Kosmos provides an alternative to a classic [Kubernetes Kapsule](/con
...
+    * Configure **pool options**, including node count and whether to enable autoscaling. Options also include enabling autoheal and linking to a placement group, or you can retain default settings.
+        <Message type="tip">
+          Unsure about the autoheal feature? [Learn more about autoheal.](/containers/kubernetes/concepts/#autoheal)
+        </Message>
+    * Enable full isolation if required.
+        <Message type="tip">
+          Need more information on full isolation? [Learn more about full isolation.](/containers/kubernetes/reference-content/secure-cluster-with-private-network/#what-is-the-difference-between-controlled-isolation-and-full-isolation)
+        </Message>
+  * For **multi-cloud** pools:
+    * TBD.
```
Comment (jcirinosclwy) [inline containers/kubernetes/how-to/create-kosmos-cluster.mdx:73]: Is this supposed to be here? (just checking)

File: containers/kubernetes/quickstart.mdx [hunk 9]
```diff
@@ -65,29 +65,64 @@ Scaleway Kubernetes [Kapsule](/containers/kubernetes/concepts/#kubernetes-kapsul
...
-4. Configure the **Private Network** for the cluster. A Private Network allows your cluster to communicate in an isolated and secure network. Each cluster is autoconfigured using a /22 IP subnet.
-   You can either:
-    * Attach an existing Private Network (VPC) within the same Availability Zone from the drop-down.
+    * Choose the geographical **region** for the cluster.
+    * Select the control plane offer for your cluster. Options include shared or dedicated control planes.
+        <Message type="tip">
+          Need help deciding on a control plane offer? Learn more about our [Kubernetes control plane offers](/containers/kubernetes/reference-content/kubernetes-control-plane-offers/).
+        </Message>
+    * Specify the Kubernetes **version** for your cluster.
+4. Provide a **name** for the cluster, along with an optional description and tags for better organization.
```
Comment (jcirinosclwy) [inline containers/kubernetes/quickstart.mdx:87]:
```suggestion
4. Provide a **name** for the cluster. Optionally, you can add a description and tags for better organization.
```

File: containers/kubernetes/quickstart.mdx [hunk 10]
```diff
@@ -65,29 +65,64 @@ Scaleway Kubernetes [Kapsule](/containers/kubernetes/concepts/#kubernetes-kapsul
...
+This section outlines the settings for your cluster pools.
+
+1. Specify the following for each pool:
+    * The **Availability Zone** for the pool's nodes.
+    * The **node type** for the pool.
+    * Configure **pool options**, including node count and whether to enable autoscaling. Options also include enabling autoheal and linking to a placement group, or you can retain default settings.
+        <Message type="tip">
+          Unsure about the autoheal feature? [Learn more about autoheal.](/containers/kubernetes/concepts/#autoheal)
+        </Message>
+    * Enable full isolation if required.
```
Comment (jcirinosclwy) [inline containers/kubernetes/quickstart.mdx:108]:
```suggestion
    * Enable full isolation, if required.
```

File: containers/kubernetes/quickstart.mdx [hunk 11]
```diff
@@ -65,29 +65,64 @@ Scaleway Kubernetes [Kapsule](/containers/kubernetes/concepts/#kubernetes-kapsul
...
+        <Message type="tip">
+          Need more information on full isolation? [Learn more about full isolation.](/containers/kubernetes/reference-content/secure-cluster-with-private-network/#what-is-the-difference-between-controlled-isolation-and-full-isolation)
+        </Message>
+2. Click **Add pool** to integrate the pool into the cluster.
+3. To add additional pools, click **Expand** and repeat the above steps.
```
Comment (jcirinosclwy) [inline containers/kubernetes/quickstart.mdx:113]:
```suggestion
3. To add additional pools, click **Expand** and repeat the steps above.
```

File: containers/kubernetes/how-to/create-cluster.mdx [hunk 12]
```diff
@@ -1,53 +1,88 @@
...
+        <Message type="tip">
+          Need more information on full isolation? [Learn more about full isolation.](/containers/kubernetes/reference-content/secure-cluster-with-private-network/#what-is-the-difference-between-controlled-isolation-and-full-isolation)
+        </Message>
+2. Click **Add pool** to integrate the pool into the cluster.
+3. To add additional pools, click **Expand** and repeat the above steps.
```
Comment (jcirinosclwy) [inline containers/kubernetes/how-to/create-cluster.mdx:76]:
```suggestion
3. To add more pools, click **Expand** and repeat the steps above.
```

File: containers/kubernetes/how-to/create-kosmos-cluster.mdx [hunk 13]
```diff
@@ -23,24 +23,66 @@ Kubernetes Kosmos provides an alternative to a classic [Kubernetes Kapsule](/con
...
+2. Click **Add pool** to integrate the pool into the cluster.
+
+3. To add additional pools, click **Expand** and repeat the above steps.
```
Comment (jcirinosclwy) [inline containers/kubernetes/how-to/create-kosmos-cluster.mdx:76]:
```suggestion
3. To add more pools, click **Expand** and repeat the steps above.
```

**Review by jcirinosclwy — APPROVED** [review]

File: containers/kubernetes/how-to/create-cluster.mdx [hunk 14]
```diff
@@ -1,53 +1,92 @@
...
+
+## Pool configuration
+
+This section outlines the settings for your cluster pools. You can configure as many pools for your cluster as you require.
+
+1. Configure the following for each pool:
+    * Choose an **Availability Zone** for the pool's nodes.
+    * Select the **node type** for the pool.
+        <Message type="tip">
+          Need advice in choosing a node type? [Learn more about Kubernetes nodes.](/containers/kubernetes/concepts/#node)
```
Comment (RoRoJ) [inline containers/kubernetes/how-to/create-cluster.mdx:67]:
```suggestion
          Need advice on choosing a node type? [Learn more about Kubernetes nodes.](/containers/kubernetes/concepts/#node)
```

File: containers/kubernetes/quickstart.mdx [hunk 15]
```diff
@@ -65,29 +65,72 @@ Scaleway Kubernetes [Kapsule](/containers/kubernetes/concepts/#kubernetes-kapsul
...
+
+### Pool configuration
+
+This section outlines the settings for your cluster pools. You can configure as many pools for your cluster as you require.
+
+1. Configure the following for each pool:
+    * Choose an **Availability Zone** for the pool's nodes.
+    * Select the **node type** for the pool.
+        <Message type="tip">
+          Need advice in choosing a node type? [Learn more about Kubernetes nodes.](/containers/kubernetes/concepts/#node)
```
Comment (RoRoJ) [inline containers/kubernetes/quickstart.mdx:108]:
```suggestion
          Need advice on choosing a node type? [Learn more about Kubernetes nodes.](/containers/kubernetes/concepts/#node)
```

**Review by RoRoJ — APPROVED** [review]

File: containers/kubernetes/how-to/create-cluster.mdx [hunk 16]
```diff
@@ -1,53 +1,92 @@
...
+  description: Learn how to create and manage a Scaleway Kubernetes Kapsule cluster, an advanced managed environment for containerized applications.
 content:
   h1: How to create a Kubernetes Kapsule cluster
-  paragraph: Learn how to create and manage a Scaleway Kubernetes Kapsule cluster, a managed environment for running containerized applications.
-tags: kubernetes kapsule-cluster kapsule
+  paragraph: Master the process of creating and managing a Scaleway Kubernetes Kapsule cluster, an advanced managed environment for containerized applications.
+tags: kubernetes, kapsule-cluster, kapsule
 dates:
-  validation: 2024-06-17
+  validation: 2024-08-08
```
Comment (bene2k1) [inline containers/kubernetes/how-to/create-cluster.mdx:11]:
```suggestion
  validation: 2024-09-30
```

File: containers/kubernetes/how-to/create-kosmos-cluster.mdx [hunk 17]
```diff
@@ -1,19 +1,20 @@
...
   title: How to create a Kubernetes Kosmos cluster
   description: Learn how to create a Kubernetes Kosmos cluster with managed Scaleway Instances and external nodes from any cloud provider.
 content:
   h1: How to create a Kubernetes Kosmos cluster
   paragraph: Learn how to create a Kubernetes Kosmos cluster with managed Scaleway Instances and external nodes from any cloud provider.
-tags: kubernetes kosmos kosmos-cluster cluster
+tags: kubernetes, kosmos, kosmos-cluster, cluster
 dates:
-  validation: 2024-06-17
+  validation: 2024-08-08
```
Comment (bene2k1) [inline containers/kubernetes/how-to/create-kosmos-cluster.mdx:11]:
```suggestion
  validation: 2024-09-30
```

File: containers/kubernetes/quickstart.mdx [hunk 18]
```diff
@@ -7,7 +7,7 @@ content:
   paragraph: This page shows you how to get started with Scaleway Kubernetes Kapsule and Kosmos.
 tags: kubernetes cluster multi-cloud pool
 dates:
-  validation: 2024-07-02
+  validation: 2024-08-08
```
Comment (bene2k1) [inline containers/kubernetes/quickstart.mdx:10]:
```suggestion
  validation: 2024-09-30
```

**Pull request merged**